frontend/src/scenes/max/maxGlobalLogic.tsx

```diff
@@ -17,14 +17,58 @@ import { TOOL_DEFINITIONS, ToolRegistration } from './max-constants'
 import type { maxGlobalLogicType } from './maxGlobalLogicType'
 import { maxLogic } from './maxLogic'
 
+/**
+ * Build available pages context with descriptions for the navigate tool
+ */
+function buildSceneDescriptionsContext(): string {
+    const pageEntries: string[] = []
+
+    // Get all navigate URL keys from urls object that are functions
+    const navigateUrlKeys = Object.keys(urls).filter((key) => typeof urls[key as keyof typeof urls] === 'function')
+
+    for (const urlKey of navigateUrlKeys) {
+        try {
+            // Call the URL function with nothing to get the base URL route
+            const url = urls[pageKey as AssistantNavigateUrls]()
```

Comment:
Shouldn't we iterate only over those which are `AssistantNavigateUrls` rather than all of them? Aren't we including more stuff in the context than what Max can actually do?